crates/ruff/src/rules/flake8_pyi/rules.rs

```diff
@@ -26,15 +26,15 @@ impl fmt::Display for VarKind {
 }
 
 define_violation!(
-    /// ### What it does
+    /// ## What it does
```

Comment:
Do you mind separating this change into a separate PR, to make this one more focused and easier to review?

Reply:
See #2749.

Reply:
Will also take [Fix some code block markers](https://github.com/charliermarsh/ruff/pull/2747/commits/aecbc08ddb9da34188fe0f8d67b24a47018ac217) over there too to avoid conflicts here...